app/src/main/java/org/greenstand/android/TreeTracker/activities/CameraActivity.kt

```diff
@@ -328,6 +434,98 @@ class CameraActivity : AppCompatActivity(), Camera.PictureCallback, View.OnClick
     }
 
 
+/** UNIT TEST CODE FOR FOCUS METRIC
+    private fun printImage(image: Array<Array<Int>> ) {
```

Comment:
What do you think of moving this testing code into its own file?

Reply:
I move everything to ImageUtils